Tests/StreamChatTests/StreamChatFlakyTests.xctestplan

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "defaultOptions" : {
-    "defaultTestExecutionTimeAllowance" : 60,
+    "defaultTestExecutionTimeAllowance" : 300,
```

Comment:
is that 300 seconds?

Reply:
Maybe I went too far with it and should reduce it to 3 minutes. There is an existing performance test which rans around 80 seconds currently.